src/components/Button/Button.spec.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import Button from './Button';
+
+describe('Button component', () => {
+	it('shold render children', () => {
```

Comment:
Found a little typo 🙂 

```suggestion
	it('should render children', () => {
```